Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -3700,7 +3700,7 @@ + (void)resurrectNewOutgoingMessages2:(NSUInteger)count
                             expiresInSeconds:(configuration.isEnabled ? configuration.durationSeconds
                                                                       : 0)expireStartedAt:0
                               isVoiceMessage:NO
-                            groupMetaMessage:TSGroupMessageNone
+                            groupMetaMessage:TSGroupMessageUnspecified
```

Comment:
This is just a rename to (hopefully) clarify the semantics of this value a little bit.

It doesn't mean that it's *not* a group message, it means we've declared no "special" group message.